### ISSUE TITLE
Add pip check step

### ISSUE THREAD

**Issue by ZLLentz** [pcdshub/pcds-envs#175]
Should probably do this per-repo as well, but this will help us find repos with inconsistent dependencies.

**Comment by ZLLentz** [comment]
Current output:
```
$ pip check
psdm-qs-cli 0.3.1 requires openpyxl, which is not installed.
databroker 1.2.0 requires bluesky-live, which is not installed.
```

**Comment by ZLLentz** [comment]
closed by #176
